code/modules/paperwork/paper.dm

```diff
@@ -298,6 +311,31 @@
 
 		user << "<span class='notice'>You stamp the paper with your rubber stamp.</span>"
 
+	if(is_hot(P))
+		if(user.disabilities & CLUMSY && prob(10))
+			user.visible_message("<span class='warning'>[user] accidentally ignites themselves!</span>", \
+								"<span class='userdanger'>You miss the paper and accidentally light yourself on fire!</span>")
+			user.unEquip(P)
+			user.adjust_fire_stacks(1)
+			user.IgniteMob()
+			return
+
+		if(!(in_range(user, src))) //to prevent issues as a result of telepathically lighting a paper
+			return
+
+		burning = 1
+		user.unEquip(src) //Without this problems can arise, i.e. ashes in a backpack
+		icon_state = "paper_onfire"
+		info = "[stars(info)]" //kinda hard to read a burning paper

```

Comment:
I figured that since the paper was _on fire_ it would be hard to read. I could maybe do a different approach. :fire: